tests/template/elegant_test.py

```diff
@@ -42,8 +42,10 @@ def _elegant():
 
 def _expr(expr, expect, variables=None):
     from pykern.pkunit import pkok, pkfail
-    res = _elegant().get_application_data(PKDict(
+    from sirepo.template import template_common
+    res = template_common.stateful_compute_dispatch(PKDict(
         method='rpn_value',
+        simulationType='elegant',
         value=expr,
         variables=variables or {},
```

Comment:
Change to PKDict() when you touch code like this.

Reply:
e3c31a2ed